src/utils/feature.flag.ts

```diff
@@ -2,14 +2,8 @@
  * Feature flags will be determined by environment variables and all environment variables in nodejs are
  * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
```

Comment:
Worth updating this comment I think to match the new implementation - so saying something like "'1', 'true' and 'on' indicate that a feature is active, any other value means it is not".

Reply:
fixed